tests/serial/watch-integral/WatchIntegralScaleAndNoScale.cpp

```diff
@@ -76,26 +76,26 @@ BOOST_AUTO_TEST_CASE(WatchIntegralScaleAndNoScale)
     // Initialize the mesh
     double dt = interface.initialize();
 
-    int    dataAID = interface.getDataID("DataTwo", meshTwoID);
+    int    dataOneID = interface.getDataID("DataOne", meshTwoID);
     double valueA, valueB, valueC;
 
     while (interface.isCouplingOngoing()) {
 
-      interface.readScalarData(dataAID, idA, valueA);
-      interface.readScalarData(dataAID, idB, valueB);
-      interface.readScalarData(dataAID, idC, valueC);
+      interface.readScalarData(dataOneID, idA, valueA);
+      interface.readScalarData(dataOneID, idB, valueB);
+      interface.readScalarData(dataOneID, idC, valueC);
 
       dt = interface.advance(dt);
     }
     interface.finalize();
 
     {
       std::string fileName = "precice-SolverTwo-watchintegral-WatchIntegral.log";
-      auto        result   = readDoublesFromTXTFile(fileName, 4);
+      auto        result   = readDoublesFromTXTFile(fileName, 3);
```

Comment:
Magic number? I don't understand what this is doing.

Reply:
It is skipping the first line of the log file. You could maybe call the variable `numberOfEntriesToSkip`.

Reply:
Ok. Makes sense. 3 columns * 1 line :+1: